pom.xml

```diff
@@ -172,13 +192,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
-            <version>1.28</version>
+            <version>1.34</version>
```

Comment:
It wasn't out when you cut this, but I think we really, really want 1.35 since it patches some nasty memory leaks in third-party plugins (outside of pipeline itself, but pipeline tends to get blamed if people don't know the root cause).

Reply:
Works for me.